pkg/driver/mount.go

```diff
@@ -198,7 +178,7 @@ func (pml *ProcMountLister) ListMounts() ([]mount.MountPoint, error) {
 }
 
 func (m *S3Mounter) IsMountPoint(target string) (bool, error) {
-	if _, err := m.Fs.Stat(target); os.IsNotExist(err) {
+	if _, err := os.Stat(target); os.IsNotExist(err) {
```

Comment:
I prefer creating a temporary folder and use real OS operations instead of mocking them. It was failing with the addition of new method and I just replaced with temporary folder instead of passing this FS interface to new `CreateAWSProfile`/`CleanupAWSProfile` functions.